webapp/src/App.js

```diff
@@ -78,53 +84,63 @@ class DecksList extends React.Component {
 
 class DecksListDeck extends React.Component {
   proptypes = {
-    deck: PropTypes.object,
-    api: PropTypes.instanceOf(Api),
```

Comment:
None of the child components of `DecksList` need to be aware of the API's existence now.